README.md

```diff
@@ -38,8 +38,9 @@ We are currently in Beta stage.  More features and improvements are coming.
 
 ## News
 
-v0.3 has just been released! Huge performance improvement (up to 19x). See release note
-[here](https://github.com/dmlc/dgl/releases/tag/v0.3).
+v0.4 has just been released! DGL now support **heterogeneous graphs**, and comes
+with a subpackage **DGL-KE** that computes embeddings for large knowledge graphs.
```

Comment:
How large? 

[PyTorch-BigGraph](https://github.com/facebookresearch/PyTorch-BigGraph#) claims to support "graphs with up to billions of entities and trillions of edges" and experimented with ["the full Freebase dataset, which contains over 100 million nodes and 2 billion edges"](https://www.sysml.cc/doc/2019/71.pdf)

Reply:
Our team has trained dgl-ke [on freebase](https://github.com/dmlc/dgl/tree/master/apps/kg#performance), the number of triples(edges) is 1.9 billion, the same as pbg's.

Reply:
The [datasets](https://github.com/dmlc/dgl/tree/master/apps/kg#datasets) statistics are absent.